### PR TITLE
[Rule Tuning] Keychain Password Retrieval via Command Line

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/991

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml [hunk 1]
```diff
@@ -26,11 +26,13 @@ risk_score = 73
...
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:security and process.args:("find-generic-password" or "find-internet-password")
+process where event.type == "start" and
+ process.name:"security" and process.args : "-wa" and process.args:("find-generic-password", "find-internet-password") and
+ process.args : ("Chrome*", "Chromium", "Opera", "Safari*", "Brave", "Microsoft Edge", "Edge", "Firefox*") and
+ not process.parent.executable : "/Applications/Keeper Password Manager.app/Contents/Frameworks/Keeper Password Manager Helper*/Contents/MacOS/Keeper Password Manager Helper*"
```
Comment (brokensound77) [inline rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:35]: nit for consistency:

```suggestion
process where event.type == "start" and
 process.name : "security" and process.args : "-wa" and process.args : ("find-generic-password", "find-internet-password") and
 process.args : ("Chrome*", "Chromium", "Opera", "Safari*", "Brave", "Microsoft Edge", "Edge", "Firefox*") and
 not process.parent.executable : "/Applications/Keeper Password Manager.app/Contents/Frameworks/Keeper Password Manager Helper*/Contents/MacOS/Keeper Password Manager Helper*"
```

**Review by brokensound77 — APPROVED** [review]
LGTM 🚀

File: rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml [hunk 2]
```diff
@@ -10,10 +10,10 @@ Adversaries may collect keychain storage data from a system to in order to acqui
 built-in way for macOS to keep track of users' passwords and credentials for many services and features, including Wi-Fi
 and website passwords, secure notes, certificates, and Kerberos.
 """
-false_positives = ["Trusted parent processes accessing their respective application passwords."]
+false_positives = ["Applications for Passwords Management."]
```
Comment (bm11100) [inline rules/macos/credential_access_keychain_pwd_retrieval_security_cmd.toml:13]:
```suggestion
false_positives = ["Applications for password management."]
```

**Review by bm11100 — APPROVED** [review]
🛫

**Pull request merged**